easybuild/easyconfigs/l/LASTZ/LASTZ-1.02.00_Makefile.patch

```diff
@@ -0,0 +1,12 @@
+--- lastz-distrib-1.02.00/src/Makefile	2010-01-13 18:00:09.000000000 +0100
```

Comment:
@RvDijk please include a line of comment describing what the patch does and why it is needed, and mention yourself as an author:

```
remove use of -Werror because ...
author: Ruben van Dijk (<affiliation>)
```